### PR TITLE
Add logs for device injector

### PR DESCRIPTION
**Pull request by Jiaqicao257**

### CODE REVIEW

File: nri_device_injector/nri_device_injector.go [hunk 1]
```diff
@@ -99,18 +99,21 @@ func (p *plugin) CreateContainer(_ context.Context, pod *api.PodSandbox, contain
 	l.Info("Started CreateContainer")
 	devices, err = getDevices(ctrName, pod.Annotations)
 	if err != nil {
+		l.Warnf("Failed to get device from pod annotation: %v", err)
```
Comment (samuelkarp) [inline nri_device_injector/nri_device_injector.go:102]:
```suggestion
		l.WithError(err).Warn("Failed to get device from pod annotation")
```

File: nri_device_injector/nri_device_injector.go [hunk 2]
```diff
@@ -99,18 +99,21 @@ func (p *plugin) CreateContainer(_ context.Context, pod *api.PodSandbox, contain
...
 		return nil, nil, err
 	}
 	adjust := &api.ContainerAdjustment{}
 
 	if len(devices) == 0 {
 		l.Debug("No devices annotated...")
+		l.Info("Finished CreateContainer")
```
Comment (samuelkarp) [inline nri_device_injector/nri_device_injector.go:109]: If we want this kind of log, maybe put it at the top of the function with a `defer`?

File: nri_device_injector/nri_device_injector.go [hunk 3]
```diff
@@ -99,18 +99,21 @@ func (p *plugin) CreateContainer(_ context.Context, pod *api.PodSandbox, contain
...
 		return adjust, nil, nil
 	}
 	for _, d := range devices {
 		l.WithField("device", d.Path).Info("Annotated device")
 		deviceNRI, err := d.toNRIDevice()
 		if err != nil {
+			l.WithField("device", d.Path).Warnf("Failed to get device from path: %v", err)
```
Comment (samuelkarp) [inline nri_device_injector/nri_device_injector.go:116]:
```suggestion
			l.WithField("device", d.Path).WithError(err).Warn("Failed to get device from path")
```

File: nri_device_injector/nri_device_injector.go (hunk 1 above)
Comment (Jiaqicao257) [reply nri_device_injector/nri_device_injector.go]: Done.

File: nri_device_injector/nri_device_injector.go (hunk 3 above)
Comment (Jiaqicao257) [reply nri_device_injector/nri_device_injector.go]: Done.

File: nri_device_injector/nri_device_injector.go (hunk 2 above)
Comment (Jiaqicao257) [reply nri_device_injector/nri_device_injector.go]: Done, thanks for the suggestion!

**Review by samuelkarp — APPROVED** [review]

**Pull request merged**